src/components/QuoteMachine.css

```diff
@@ -0,0 +1,213 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
it might be a better practice to have only the styles for QuoteMachine in this file and general styling in another file. Also consider following the BEM standard

Reply:
Hi @pesinasiller 
Thanks so much for your feedback! 😊 I’ll separate the QuoteMachine styles and general styles into different files, and start using BEM for better organization. I’m excited to dive deeper into BEM during our one-to-one meeting. Thanks again! 👍